scripts/install-deps.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+apt-get update -qq && \
+apt-get install -y \
+        gdal-bin \
+        python-gdal \
+        pandoc \
```

Comment:
Had to include this since: 
![image](https://user-images.githubusercontent.com/19597045/79283997-d825cb00-7e8f-11ea-8a0e-da779c96e9b9.png)
- https://travis-ci.com/github/publiclab/mapknitter-exporter-sinatra/jobs/319424125

Reply:
Hi,
In second review I still see that you've introduced `pandoc` as dependency. In previous version `kramdown` did the exact same and pandoc is just an optional dependency (which is very large). Please try again without including pandoc, I took care that it wasn't required, unless I am wrong. Thanks for the perseverance!
```suggestion
        pandoc \
```

Reply:
@icarito When I remove pandoc I get the error:
![image](https://user-images.githubusercontent.com/19597045/83978323-0b5b6800-a8dd-11ea-9192-67ee07c13f21.png)

Via: https://travis-ci.com/github/publiclab/mapknitter-exporter-sinatra/jobs/345621753

Reply:
Should I revert 3428c40075a3c12371b8948f76cf7192296e844c to get this merged?